modules/Template/PortalPublication.yaml

```diff
@@ -0,0 +1,70 @@
+#######################################
+# Portal          
+#######################################
+
+# Note: this template definition uses the alternate syntax with 'attribute'
+
+classes:
+  PublicationTemplate:
+    is_a: Template
+    description: >
+      A generalized data model template for Publications to be hopefully used across dccs
+    notes:
+    - This structure is based on a CSV file from MC2 annotations with customized fields as attributes.
+    attributes:
+      grantNumber:
+        range: string
+        required: true
+      consortiumName:
+        range: string
+        required: true
+      themeName:
+        range: string
+        required: true
+      doi:
+        range: string
+        required: true
+      journal:
+        range: string
+        required: true
+      pmid:
+        range: integer
+        required: true
+      pubmedUrl:
+        range: string
+        required: true
+      title:
+        range: string
+        required: true
+      year:
+        range: integer
+        required: true
+      diseaseFocus:
+        range: string
+        multivalued: true
+        required: false
+      authors:
+        range: string
+        multivalued: true
+        required: false
+      abstract:
+        range: string
+        required: true
+      assay:
+        range: string
+        multivalued: true
+        required: false
+      accessibility:
+        range: string
+        required: true
+
+  NFPublication:
+    is_a: PublicationTemplate
+    description: A slightly more specialized publication concept intended for the specific scope of the NF Portal.
+    attributes:
+      studyUse:
```

Comment:
I think instead of leaving this as string, we create an enum called "StudyProductTypeEnum" (or something else more fitting if you can think of it) with values "primary" and "secondary". Use the example enum syntax for Sex, in https://github.com/nf-osi/nf-metadata-dictionary/blob/main/modules/Sample/Sex.yaml. For now, just put the new enum as a file in https://github.com/nf-osi/nf-metadata-dictionary/tree/main/modules/Other. Then we can reuse this enum later on if we need.

Reply:
Definitely prefer to make it an enum!